config/api.yml

```diff
@@ -473,6 +473,29 @@
       :get:
       - :name: read
         :identifier: miq_cloud_networks_view
+  :cloud_volumes:
+    :description: Cloud Volumes
+    :identifier: cloud_volume
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
is this needed a primary collection (i.e. I want to query all cloud volumes) or as a subcollection (I need to query cloud volumes pertaining to a CI) or both ?